src/router/router.tsx

```diff
@@ -0,0 +1,24 @@
+import { createBrowserRouter } from 'react-router-dom';
+import { Root } from './root';
+import { HomePage } from '../pages/home-page/home-page';
+import { NotFoundPage } from '../pages/notfound-page/notfound-page';
+import { PagePath } from './enums';
+import { LoginPage } from '../pages/login-page/login-page';
+import { RegistrationPage } from '../pages/registration-page/registration-page';
+
+export const router = createBrowserRouter([
+  {
+    path: PagePath.root,
+    Component: Root,
+    children: [
+      { index: true, Component: HomePage },
+      { path: PagePath.notFound, Component: NotFoundPage },
+      { path: PagePath.loginPage, Component: LoginPage },
+      { path: PagePath.registrationPage, Component: RegistrationPage },
+    ],
+  },
+  {
+    path: PagePath.notFound,
+    Component: NotFoundPage,
+  },
```

Comment:
Попробуй сделать 2 лейаута: один с хэдером для авторизированных, второй для неавторизированных без хэдера
```suggestion
  {
    element: <GuestLayout />,
    children: [
      // { index: Path.aboutPage, element: <AboutPage /> },
      { path: PagePath.loginPage, element: <LoginPage /> },
      { path: PagePath.registrationPage, element: <RegistrationPage /> },
    ],
  },
  {
    element: <AuthLayout />,
    children: [
      { path: PagePath.root, element: <HomePage /> },
      // { path: PagePath.profile, element: <ProfilePage /> },
    ],
  },
  { path: '*', Component: NotFoundPage },
```